zb-classifier.js

```diff
@@ -403,6 +416,63 @@ class ZigbeeClassifier {
     );
   }
 
+  addButtonOnProperty(node, genOnOffOutputEndpoint) {
+    const property = this.addProperty(
+      node,                           // device
+      'on',                           // name
+      {                               // property description
+        '@type': 'BooleanProperty',
```

Comment:
Couldn't this be `PushedProperty`?

Reply:
It could, but it didn't feel right given the way that the switch is used (see below)

Reply:
Ah, ok. I didn't know what the hardware actually was.